README.md

```diff
@@ -135,6 +135,8 @@ Schemer\Formatter::text()
 
 ### `Schemer\Validator`
 
+Some methods take an optional custom error `string $error`
```

Comment:
My only issue is that I'd probably pad this out a bit. "For many of these methods, a custom `$error` string can be supplied as the final parameter to override the Schemer default" or something like that. Really just a preference thing... sounds more... flamboyant :star: